notecard/main.go

```diff
@@ -16,6 +16,8 @@ const (
 	I2C    = "i2c"
 )
 
+const SerialDevice = "/dev/tty.usbmodemNOTE1"
```

Comment:
maybe "defaultSerialDevice" (to indicate it's a default, and lowercase because it does not need to be exported).
No big deal since we're in a main though.

Reply:
good point, better to be clear